classes/submission/Submission.inc.php

```diff
@@ -248,6 +248,50 @@ function setHideAuthor($hideAuthor) {
 	}
 
 	/**
+	 * Set comments to editor.
+	 * @param $commentsToEditor string
+	 */
+	function setCommentsToEditor($commentsToEditor) {
```

Comment:
hmmm, seems to be a left over. Maybe a rebase didn't work here 🤔  ... Removed it.